OsmAnd/src/net/osmand/plus/utils/AndroidUtils.java

```diff
@@ -1503,4 +1505,18 @@ public static int[] getRelativeMargins(@NonNull View parentView, @NonNull View v
 	public static OsmandApplication getApp(@NonNull Context context) {
 		return ((OsmandApplication) context.getApplicationContext());
 	}
+
+	public static void registerBroadCastReceiver(@NonNull Activity activity, @NonNull String action, @NonNull BroadcastReceiver receiver) {
+		registerBroadCastReceiver(activity, action, receiver, false);
+	}
+
+	public static void registerBroadCastReceiver(@NonNull Activity activity, @NonNull String action, @NonNull BroadcastReceiver receiver, boolean export) {
+		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+			// For API level 33 and above
+			activity.registerReceiver(receiver, new IntentFilter(action), export ? Context.RECEIVER_EXPORTED : Context.RECEIVER_NOT_EXPORTED);
+		} else {
+			// For older API levels
+			activity.registerReceiver(receiver, new IntentFilter(action));
+		}
+	}
```

Comment:
We have 10 other places where it could be reused. Also, the comments look unnecessary